parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -317,6 +331,26 @@ private static int getMaxPaddingSize(Configuration conf) {
     return conf.getInt(MAX_PADDING_BYTES, DEFAULT_MAX_PADDING_SIZE);
   }
 
+  public static ValuesWriterFactory getValuesWriterFactory(Configuration conf) {
+    Class<? extends ValuesWriterFactory> factoryOverride =
```

Comment:
Use this instead to avoid duplicating a Configuration method:

```java
    Class<? extends ValuesWriterFactory> factoryOverride = conf.getClass(
        WRITER_FACTORY_OVERRIDE, DefaultValuesWriterFactory.class,
        ValuesWriterFactory.class);
```

Then you can get rid of `ConfigurationUtil`.